src/xdp/program.c

```diff
@@ -588,6 +588,20 @@ XdpProgramTrace(
                 Program, i, ntohs(Rule->Pattern.NextHeader));
             break;
 
+        case XDP_MATCH_ICMPV4_ECHO_REPLY_IP_DST_MASK:
+            TraceInfo(
+                TRACE_CORE,
+                "Program=%p Rule[%u]=XDP_MATCH_ICMPV4_ECHO_REPLY_IP_DST_MASK Ip=%!IPADDR!",
```

Comment:
We need to trace both the address and the mask.

Reply:
The name of this rule is `DST_MSK` implying the mask is part of the rule, and the field being used is also `Rule.Pattern.IpMask` again implying the use of a mask, so it would be internally consistent to support a configurable mask.

That said, the issue doesn't mention masking at all, and so I think not masking would be fine, but we'd need to avoid any references to masks.